### PR TITLE
Transactions api search by targetid

### PR DESCRIPTION
**Pull request by HHjolany**
## Link to Click Up ticket

[https://app.clickup.com/t/1mufk2n](https://app.clickup.com/t/1mufk2n)

### *What is the problem we're trying to solve*

With regard to adding search by targetId(tenure id) in transaction endpoint, relevant unit test and integration test added too

#### _Checklist_

- [x] Search by target id in transaction endpoint
- [x] Unit test
- [x] Integration test

### CODE REVIEW

**Review by EdwardAnoghena — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,55 +52,11 @@ commands:
...
-          name: plan
-          command: |
-            cd ./terraform/<<parameters.environment>>/
-            terraform plan -out=plan.out
-      - persist_to_workspace:
-          root: *workspace_root
-          paths:
-            - .aws
-            - project/*
-  terraform-compliance:
```
Comment (LBHMGeorgieva) [inline .circleci/config.yml:65]: Why is this removed?
